test/project.e2e-spec.ts

```diff
@@ -298,6 +298,148 @@ describe('Project e2e', () => {
     );
   });
 
+  it('List of projects sorted by name to be alphabetical, ignoring case sensitivity. Order: ASCENDING', async () => {
+    await createUser(app, { displayFirstName: 'Tammy' });
+    //Create three projects with mixed cases.
+    await createProject(app, {
+      name: 'a project 2' + faker.random.uuid(),
+      type: ProjectType.Translation,
+    });
+    await createProject(app, {
+      name: 'Another project 2' + faker.random.uuid(),
+      type: ProjectType.Translation,
+    });
+    await createProject(app, {
+      name: 'Big project 2' + faker.random.uuid(),
+      type: ProjectType.Translation,
+    });
+    await createProject(app, {
+      name: 'big project also 2' + faker.random.uuid(),
+      type: ProjectType.Translation,
+    });
+    const sortBy = 'name';
+    const ascOrder = 'ASC';
+    const { projects } = await app.graphql.query(
+      gql`
+        query projects($input: ProjectListInput!) {
+          projects(input: $input) {
+            hasMore
+            total
+            items {
+              id
+              name {
+                value
+              }
+            }
+          }
+        }
+      `,
+      {
+        input: {
+          sort: sortBy,
+          order: ascOrder,
+          filter: {
+            mine: true,
+          },
+        },
+      }
+    );
+    // Set a flag that's going to indicate if the projects are in order
+    const isAscending = true;
+    const items = projects.items;
+    const sorted = orderBy(items, (proj) => proj.name.value.toLowerCase(), [
+      'asc',
+    ]);
+    expect(sorted).toEqual(items);
+    expect(isAscending).toBe(true);
+
+    //delete all projects
+    await Promise.all(
+      projects.items.map(async (item: { id: any }) => {
```

Comment:
Are we sure we want to delete all projects after this? What about just deleting the ones that were created above in this test case?

Reply:
it is deleting only the ones created. It's going through the projects.items and deleting only the projects with those ids (which it's getting from `projects` ) if you look above, you'll see that we created/registered a user named Tammy, and that user created four projects. when that user lists the projects, only the projects that Tammy created will be listed, and she can delete them.
.......
..... unless I totally don't understand how permissions with that user work, which is a huge possibility ;)

Reply:
> when that user lists the projects, only the projects that Tammy created will be listed,

That's only true if the `mine` filter was implemented, which it's not right now. Actually permissions also affect it, which I don't fully understand either. So maybe it's ok.